Bixal/move.mil
# Separate MapsJs and Geo/Dist keys.

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist

I have…

      against my changes
## Summary of Changes

This pull request…

- Separate GoogleKeys MapsJS and Geo/Dist to add restrictions on the API calls.

## Testing

To verify the changes proposed in this pull request…

1. Use locator maps and ppm estimator
2. Make sure the map is working correctly
